src/schema/objects/metadata.yaml

```diff
@@ -1022,7 +1022,15 @@ HEDVersion:
   description: |
     If HED tags are used:
     The version of the HED schema used to validate HED tags for study.
-  type: string
+    May include a single schema or a base schema and one or more library schema.
+  anyOf:
+  - type: string
+  - type: object
+    properties:
+      base:
+        type: string
+      libraries:
+        type: object
```

Comment:
further formalization isn't possible (to e.g. the `[string]` as `list(string)`) with current schema? (I just don't know it well)

Reply:
I don't know either.... I was hoping for a suggestion.

Reply:
Sorry, what isn't expressed here that you want to be? Is the issue that `libraries` can be a list of strings in addition to an object?

Reply:
I think it is okay as is.  My question is whether object can be resolved further.  I don't think it can be from my reading of the current `yaml`, but I would appreciate confirmation.

Reply:
@tsalo -- it is more of a question on what type'ing is allowed here... I RTFMed https://github.com/bids-standard/bids-specification/tree/master/src/schema#metadatayaml to suggest now
```suggestion
        type: array
        items:
            type: string
```

Reply:
please ignore me above -- I should just let this Friday evening take over me... `object` is a json schema way to say `dict` so it was all good. And we just need `type`
```suggestion
        type: object
        additionalProperties:
            type: string
```
to limit to `string` to `string` mapping

Reply:
Ah, okay. That's clear enough. I don't think there will be any problem with that level of specification in the schema.